index.html

```diff
@@ -145,7 +150,7 @@ <h3 class="subtitle">Skills</h3>
         </section>
     </section>
 
-    <footer class="contactContainer">
+    <footer class="contactContainer" id="Contact">
```

Comment:
- [ ] Please, consider changing the font size for this paragraph! 🎨 🛠️ 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/199805372-d581e542-4503-486e-9dda-6ff706f432a4.png) | ![image](https://user-images.githubusercontent.com/77069011/199805436-456c1123-e039-447a-aec6-20ffd4356646.png) |